NitroxModel/Packets/EntityReparented.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using NitroxModel.DataStructures;
+
+namespace NitroxModel.Packets
+{
+    [Serializable]
+    public class EntityReparented : Packet
+    {
+        //make sure to update the parent
+        public NitroxId Id { get; }
```

Comment:
The comment sound to me like this id is the one of the old parent but that isn't the case, right? If it is the old parent id I would rename this field to `OldParentId`